pkgs/top-level/python-packages.nix

```diff
@@ -8751,6 +8751,8 @@ in {
     };
   };
 
+  pepper = callPackage ../development/python-modules/pepper { };
```

Comment:
... and the file also shouldn't be in `python-modules`. Please check the current version of the manual (`nix-build doc`) for the guidelines on Python packages.